### ISSUE TITLE
after gradlew clean build not release default shell due unit tests

### ISSUE THREAD

**Issue by eprogramming** [aeshell/mterm#92]

**Comment by eprogramming** [comment]
Going to double check this after 0.57 aesh-extensions release.

**Comment by eprogramming** [comment]
working fine ;)

```shell
[hf@localhost mterm]$ ./gradlew clean build
:clean
:compileJava
Note: /home/hf/dev/workspace/mterm/src/main/java/org/jboss/aesh/mterm/action/menu/MenuColorAction.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
:processResources
:classes
:jar
:assemble
:pmdMain
:pmdTest
:compileTestJava
:processTestResources
:testClasses
:test
:check
:build

BUILD SUCCESSFUL

Total time: 25.192 secs
[hf@localhost mterm]$ 
```

**Comment by eprogramming** [comment]
bug still here

**Comment by eprogramming** [comment]
this was fixed on æsh 0.58-SNAPSHOT.

**Comment by eprogramming** [comment]
fixed on æsh
